lib/connect/api/products.js

```diff
@@ -165,6 +165,74 @@ class ProductResource extends GenericResource {
     return response.json();
   }
 
+  /**
+   * Returns the *Parameter* objects configured for the parameterId.
```

Comment:
Better "Returns the *Parameter* object identified by paramId for the product identified with productId"